### ISSUE TITLE
emb-w-12 - Tabs

### ISSUE THREAD

**Issue by bertramlyons** [LibraryOfCongress/embARC#13]
emb-w-12	Tabs	Yes	2-High	

Tabs with "non-compliance" indicators are not announced as such to screen reader users.

Ensure that in addition to the visual "!" indicator, a text equivalent is provided for screen reader users, e.g. "contains non-compliant entries".
